druid-shell/src/backend/wayland/outputs/output.rs

```diff
@@ -0,0 +1,212 @@
+use super::super::display;
+use super::super::error;
+use super::super::outputs;
+use wayland_client as wlc;
+use wayland_client::protocol::wl_output;
+use wayland_client::protocol::wl_registry;
+use wayland_protocols::unstable::xdg_output::v1::client::zxdg_output_manager_v1;
+use wayland_protocols::unstable::xdg_output::v1::client::zxdg_output_v1;
+
+pub trait Consumer {
+    fn consume(
```

Comment:
this trait is no longer useful? There is only impl and just a single call to `consume`. that too in this module